docs/pipelineruns.md

```diff
@@ -645,7 +643,7 @@ Consult the documentation of the custom task that you are using to determine whe
 
 ### Mapping `ServiceAccount` credentials to `Tasks`
 
-If you require more granularity in specifying execution credentials, use the `serviceAccountNames` field to
+If you require more granularity in specifying execution credentials, use the `taskRunSpecs[].taskServiceAccountName` field to
```

Comment:
not relevant to this PR but maybe in v1 we can change this to `taskRunSpecs[].serviceAccountName`